### ISSUE TITLE
JHS generated files do not properly move into directories on Linux

### ISSUE THREAD

**Issue by Severnarch** [Severnarch/JavaHTTPServer#7]
**JavaHTTPServer Version:** `1.0.2`

**Exit Code:** `None`

**Reproduction Steps:**
1. Run `JavaHTTPServer.1.0.2.jar` on any Linux distribution
2. Exit out using `^C^Z`
3. `ls` the parent directory of JHS
4. The files generated aren't in a directory like on Windows, but are their own files with names including the Windows directory separator.
```
root@nanachi-main:~/JHS# ls /root/ 
 JHS  'JHS\configs\server.cfg'  'JHS\logs'  'JHS\logs\14-12-2024_11-51-52_GMT.txt'
 ```

**Comment by Severnarch** [comment]
This is mainly because I wasn't actually able to test it on Linux myself back when I started this due to Windows 11 crashing itself whenever I tried using WSL, but now I have a VPS from [Zorrito Hosting](https://zorritohosting.com/), I can properly test JHS on Linux.

**Comment by Severnarch** [comment]
Fixed in b373aa7a63bbbdda6fd6e052993531e7cfb8897b (`1.0.3`)
